ansible/vm_files/playbook.yml

```diff
@@ -0,0 +1,61 @@
+- name: Test Playbook
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_secret_key
+      private: yes
+      prompt: Enter your trello secret key
+
+    - name: trello_secret_token
+      private: yes
+      prompt: Enter your trello secret token
+
+  tasks:
+  - name: Install Dependencies
+    ansible.builtin.yum:
+      name:
+        - python3
+        - git
+    become: yes
+
+  - name: Install poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+    args:
+      creates: ~/.local/bin/poetry
+
+  - name: Create Directory for Todo app
+    ansible.builtin.file:
+      path: /opt/todoapp
+      state: directory
+      mode: "777"
```

Comment:
Not a big issue, but worth trying to avoid getting in the habit of `777` for everything! `755` is quite common for directories, and will just limit write access